src/containers/App.js

```diff
@@ -97,9 +93,7 @@ class App extends Component {
           {/*
             emoji && <EmojiFilter emoji={emoji} onSelect={emoji => dispatch(selectEmoji(emoji))} />
           */}
-          {mcQuestions && Object.keys(mcQuestions).map(key => (
-            <FilterableBarChart key={key} questionId={key} />
-          ))}
+          <FilterableBarChart />
```

Comment:
We now assume we are visualizing one and only one non-emoji, multiple-choice question, and select that value accordingly. Only getting one multiple choice question, and getting it by reference rather tha search, means we are not using `Array#filter` to construct a list of all relevant multiple choice questions and therefore do not need to iterate over a one-element array any longer.